scripts/create-package-json.js

```diff
@@ -16,7 +16,13 @@ packages.forEach(function(pkgName) {
   }
 
   // define the package name
-  basePkgJson.name = `@agm/${pkgName}`
+  basePkgJson.name = `@agm/${pkgName}`;
+
+  //  link main, module, es2015, typings
+  basePkgJson.main = `${pkgName}.umd.js`;
```

Comment:
LGTM

Reply:
this is actually the main reason of not supporting es5! the bundler is not finding the right files!

Reply:
💯 
check this out